pkg/git/git_test.go

```diff
@@ -80,6 +81,24 @@ func TestPlainClone(t *testing.T) {
 	})
 }
 
+func TestPlainOpenMock(t *testing.T) {
+	t.Parallel()
+	t.Run("successful clone", func(t *testing.T) {
+		t.Parallel()
+		abstractedGit := &UtilsGitMock{}
+		_, err := plainOpen("directory", abstractedGit)
+		assert.NoError(t, err)
+		assert.Equal(t, "directory", abstractedGit.path)
+	})
+
+	t.Run("error on cloning", func(t *testing.T) {
+		t.Parallel()
```

Comment:
What is the reason for the parallel statement here? There are no nested tests on that level. I think that is not needed.

Reply:
see above